src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -625,9 +632,36 @@ public void setEvent(Event target, Event editedEvent) {
         this.eventList.setEvent(target, editedEvent);
     }
 
-    // TODO: Yet to be implemented
-    //@Override
-    //public void commitEventList() {}
+    @Override
+    public void commitEventList() {
+        assert accessPointer >= 0;
+        accessSequence.subList(this.accessPointer + 1, accessSequence.size()).clear();
+        versionedEventList.commit(eventList);
+        accessSequence.add(4);
+        accessPointer += 1;
+    }
+
+    @Override
+    public void undoEventList() throws VersionedListException {
+        assert accessPointer >= 0;
+        try {
+            versionedEventList.undo();
+        } catch (VersionedListException versionedListException) {
+            throw versionedListException;
+        }
+        setEventList(versionedEventList.getCurrentEventList());
+    }
+
+    @Override
+    public void redoEventList() throws VersionedListException {
+        assert accessPointer >= 0;
+        try {
+            versionedEventList.redo();
+        } catch (VersionedListException versionedListException) {
+            throw versionedListException;
+        }
```

Comment:
I think this `catch` and `try` block is unnecessary? Since it only catches the exception and throw the exact same exception?

In addition, we already added the the exception to the method signature, so I think removing the `catch` and `try` block should be fine 😅

Reply:
Noted, thanks Michael!